addons/sys_gui/fnc_inventoryMonitorPFH.sqf

```diff
@@ -0,0 +1,54 @@
+/*
+ * Author: ACRE2Team
+ * Event handler to monitor inventory changes upon ACRE radios.
+ *
+ * Arguments:
+ * 0: Eventhandler [eventhandler, function handler] <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [acre_sys_gui_inventoryMonitorPFH, 0, []] call CBA_fnc_addPerFrameHandler
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+if (!isNull INVENTORY_DISPLAY) then {
+    TRACE_1("Registering Events", "");
+    // Hide the ItemRadio slot
+    (INVENTORY_DISPLAY displayCtrl IDC_RADIOSLOT) ctrlSetPosition [0,0,0,0];
+    (INVENTORY_DISPLAY displayCtrl IDC_RADIOSLOT) ctrlCommit 0;
+
+    /* Unused MTT...
+    (INVENTORY_DISPLAY displayCtrl IDC_FG_UNIFORM_CONTAINER) ctrlSetEventHandler["MouseButtonDown", "['uniform',_this] call acre_sys_gui_fnc_handleContextMenu"];
+    (INVENTORY_DISPLAY displayCtrl IDC_FG_VEST_CONTAINER) ctrlSetEventHandler ["MouseButtonDown", "['vest',_this] call acre_sys_gui_fnc_handleContextMenu"];
+    (INVENTORY_DISPLAY displayCtrl IDC_FG_BACKPACK_CONTAINER) ctrlSetEventHandler ["MouseButtonDown", "['backpack',_this] call acre_sys_gui_fnc_handleContextMenu"];
+    */
+
+    (INVENTORY_DISPLAY displayCtrl IDC_FG_UNIFORM_CONTAINER) ctrlSetEventHandler["LBDblClick", QUOTE([1, "uniform", _this] call FUNC(onInventoryAction))];
```

Comment:
`QUOTE([1, "uniform", _this] call FUNC(onInventoryAction))`
->
`QUOTE([1, 'uniform', _this] call FUNC(onInventoryAction))`

`"` is an escape character here.

Do for this for the entire file should fix the error I think!